website/src/utils.js

```diff
@@ -1,5 +1,6 @@
 var nodemailer = require('nodemailer');
 var nconf = require('nconf');
+nconf.file({ file: './config.json' });
```

Comment:
Thought this path would be relative to `website/src/utils.js`, but it's actually relative to the root of the directory.